modules/custom/d_commerce_product/d_commerce_product.install

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Install, update, and uninstall functions for d_commerce_product.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function d_commerce_product_install() {
```

Comment:
hmm, a co z uninstall ? wszystkie te configi się odinstalują razem z modułem wtedy automatycznie?

Reply:
@grzegorz-pietrzak-droptica weź doradź tu ciut, czy uninstall tez powinien byc i powienien odinstalowaywac te moduly ?

Reply:
IMO stosowałbym przy uninstallu taką wykładnię, że usuwamy wszystko, co bez modułu i tak nie zadziała. Jeśli wywalamy widok z produktami, to wywalamy też związane z nim facety itp.

Nie wywalamy za to rzeczy typu bloki w stopce, bo ktoś mógł sobie je zmodyfikować na swojej stronie.